internal/pkg/events/events.go

```diff
@@ -211,24 +196,9 @@ func StartEvents() {
 		}
 		serverConfig = conf.(*saconfig.EventConfiguration)
 		serverConfig.ServiceType = *fServiceType
-		if *fDebug {
-			serverConfig.Debug = true
-		}
 	} else {
-		serverConfig = &saconfig.EventConfiguration{
-			AMQP1EventURL:   *fAMQP1EventURL,
-			ElasticHostURL:  *fElasticHostURL,
-			AlertManagerURL: *fAlertManagerURL,
-			Prefetch:        *fPrefetch,
-			ServiceType:     *fServiceType,
-			API: saconfig.EventAPIConfig{
-				APIEndpointURL:  *fAPIEndpointURL,
-				AMQP1PublishURL: *fAMQP1PublishURL,
-			},
-			ResetIndex: *fResetIndex,
-			Debug:      *fDebug,
-			UniqueName: *fUniqueName,
-		}
```

Comment:
So do we want to have a default path as Paul mentioned? One example for implementation could be [1]. And fail only if the default path does not exist? Eg. something like:

```
  if _, err := os.Stat(path); err != nil && os.IsNotExist(err) {...}
```

[1] https://github.com/infrawatch/collectd-sensubility/blob/master/main/main.go#L46,L49

Reply:
Yea I'm not too worried about having this right now. I kind of went back and forth, but since we're always passing the `-config` from the Operators right now, and that our current code path doesn't allow us to remove all the flags right now (for various reasons), I think I'll just leave everything else alone for now. Thanks for the pointer though.